server/routes/makeAReferral/completion-deadline/completionDeadlinePresenter.ts

```diff
@@ -15,8 +15,9 @@ export default class CompletionDeadlinePresenter {
   readonly hint = 'For example, 27 10 2021'
 
   constructor(
-    private readonly referral: DraftReferral,
+    private readonly completionDeadline: string | null,
     private readonly intervention: Intervention,
+    readonly sentReferral: boolean | undefined = undefined,
```

Comment:
what is this used for? i see it passed in the render args in the view, but not used anywhere.

Reply:
Was going to be used to determine if to show the backLink for a sent referral. We don't have the backlink for the make a referral(draft) referral flow.

Reply:
if we are wanting to show a backlink i would just recommend passing in a backlinkArgs object or similar. see later comments on decoupling logic from templates.